<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontendDashboard</class>
 <widget class="QWidget" name="FrontendDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frontend Dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Frontend</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="componentsGroup">
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="title">
        <string>Components</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutComponents">
        <item>
         <widget class="QTreeWidget" name="componentsTree">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="mainViewGroup">
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="title">
        <string>MainView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutMainView">
        <item>
         <widget class="QTreeWidget" name="mainViewTree">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutButtons">
          <item>
           <widget class="QPushButton" name="addSectionButton">
            <property name="text">
             <string>New Section</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addColumnButton">
            <property name="text">
             <string>New Column</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="propertiesGroup">
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="title">
        <string>Properties</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutProperties">
        <item>
         <widget class="QTableWidget" name="propertiesTable">
          <property name="gridStyle">
           <enum>Qt::PenStyle::SolidLine</enum>
          </property>
          <property name="rowCount">
           <number>15</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <property name="horizontalHeaderLabels" stdset="0">
           <stringlist>
            <string>Property</string>
            <string>Value</string>
           </stringlist>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
